curvefs/proto/mds.proto

```diff
@@ -123,6 +124,25 @@ message DeleteFsResponse {
     required FSStatusCode statusCode = 1;
 }
 
+message CreatePartitionRequest {
+    required string fsName = 1;
+    required uint32 count = 2;
+}
+
+message CreatePartitionResponse {
+    required FSStatusCode statusCode = 1;
+    repeated common.PartitionInfo partitionInfoList = 2;
+}
+
+// message ListPartitionRequest {
+//     required string fsName = 2;
+// }
+//
+// message ListPartitionResponse {
+//     required FSStatusCode statusCode = 1;
+//     repeated common.PartitionInfo partitionInfoList = 2;
+// }
+
```

Comment:
message AllocateS3ChunkRequest {
    required uint32 fsId = 1;
}

message AllocateS3ChunkResponse {
    // status code, default value is FSStatusCode::UNKNOWN_ERROR
   required FSStatusCode status = 1;
   required uint64 chunkId = 2;
}